python/pyspark/sql/functions/builtin.py

```diff
@@ -253,6 +255,32 @@ def lit(col: Any) -> Column:
     +------------------+-------+-----------------+--------------------+
     |     [true, false]|     []|       [1.5, 0.1]|           [a, b, c]|
     +------------------+-------+-----------------+--------------------+
+
+    Example 7: Creating a literal column as a map from a dict.
+        if spark.sql.pyspark.inferNestedDictAsStruct is False
+
+    >>> import pyspark.sql.functions as sf
+    >>> spark.range(1).select(
+    ...    sf.lit({"a": 1, "b": 2}).alias("map_col")
+    ... ).show() # doctest: +SKIP
```

Comment:
we can enable the two doctests by applying `spark.conf.set` and `spark.conf.unset`